Scripts/build.sh

```diff
@@ -1,43 +1,72 @@
 #!/usr/bin/env bash
 #
-# Builds the beta Bitwarden iOS app, and outputs an IPA file that can be uploaded to TestFlight.
+# Builds the Bitwarden iOS app.
```

Comment:
🤔 This probably needs to be updated to remove references to build variants, and add the expected values for Build Mode (device / simulator). E.g.

```
# Builds the Bitwarden iOS app.
# If run in Simulator mode, produces an APP file for the iOS Simulator for ease of automated testing.
# If run in Device mode, produces an IPA file that can be uploaded to Testflight.
#
# Usage:
#
#   $ ./build.sh <build_mode>
#
# Where build_mode is one of:
#   - Device: Build for physical iOS devices
#   - Simulator: Build for iOS Simulator
```

Reply:
Made this change, leaving this comment open for reference.